.github/workflows/docs.yml

```diff
@@ -149,25 +154,25 @@ jobs:
           workload_identity_provider: '${{ vars.GCP_WIF }}'
           service_account: '${{ vars.GCP_FIREBASE_SA }}'
 
-      # TODO: remove this step after issue https://github.com/FirebaseExtended/action-hosting-deploy/issues/174 is fixed 
-      - run: |
+      # TODO: remove this step after issue https://github.com/FirebaseExtended/action-hosting-deploy/issues/174 is fixed
+      - name: Add $GCP_FIREBASE_SA_PATH to env
+        run: |
           # shellcheck disable=SC2002
-          echo "GCP_FIREBASE_SA=$(cat ${{ steps.auth.outputs.credentials_file_path }} | tr -d '\n')" >> "$GITHUB_ENV"
+          echo "GCP_FIREBASE_SA_PATH=$(cat ${{ steps.auth.outputs.credentials_file_path }} | tr -d '\n')" >> "$GITHUB_ENV"
 
       - name: Deploy external docs to firebase
         uses: FirebaseExtended/action-hosting-deploy@v0.7.1
         with:
           repoToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Could we try removing these repoTokens?

Reply:
Fixed with: [fix(docs): remove preview URLs from deployments](https://github.com/ZcashFoundation/zebra/pull/7339/commits/04c41e44c252f2278afbc63185bb6eafe53e14cb)